loader/cunit/cu_shp2pgsql.c

```
@@ -62,16 +74,188 @@ int clean_shp2pgsql_suite(void)
+	free_loader_config(loader_config);
+	loader_state = NULL;
+	loader_config = NULL;
+}
+
+void test_ShpLoaderTopoGeometryDefaults(void)
+{
+	SHPLOADERCONFIG *config = (SHPLOADERCONFIG*)calloc(1, sizeof(SHPLOADERCONFIG));
+	SHPLOADERSTATE *state;
+
+	set_loader_config_defaults(config);
+	CU_ASSERT_FATAL(config->encoding != NULL);
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)